frontend/www/js/omegaup/components/common/Archive.vue

```diff
@@ -48,13 +48,17 @@ import { Vue, Component, Prop } from 'vue-property-decorator';
 import T from '../../lang';
 
 @Component
-export default class CourseArchive extends Vue {
+export default class CommonArchive extends Vue {
```

Comment:
```suggestion
export default class Archive extends Vue {
```

por consistencia con el nombre del archivo.